web/src/engine/osk/src/banner/suggestionBanner.ts

```diff
@@ -170,7 +176,22 @@ export class BannerSuggestion {
 
     if(suggestion && suggestion.displayAs) {
       const rawMetrics = getTextMetrics(suggestion.displayAs, format.emSize, format.styleForFont);
-      this._textWidth = rawMetrics.width;
+      let projectedHeight = 1;
+      if(rawMetrics.fontBoundingBoxAscent) {
+        projectedHeight = rawMetrics.fontBoundingBoxAscent + rawMetrics.fontBoundingBoxDescent;
+      }
+      let ratio = Math.min(1, format.height / projectedHeight);
+      let width = rawMetrics.width;
```

Comment:
```suggestion
      const ratio = Math.min(1, format.height / projectedHeight);
      const width = rawMetrics.width;
```

Although `const width = rawMetrics.width` seems unnecessary since the value is used just once.